streaming/base/converters/dataframe_to_mds.py

```diff
@@ -202,11 +160,10 @@ def write_mds(iterator: Iterable):
             raise RuntimeError('TaskContext.get() returns None')
 
         if isinstance(mds_path, str):  # local
-            output = os.path.join(mds_path, f'{id}')
-            out_file_path = output
+            output_path = output = os.path.join(mds_path, f'{id}')
         else:
             output = (os.path.join(mds_path[0], f'{id}'), os.path.join(mds_path[1], f'{id}'))
-            out_file_path = output[0]
+            output_path = ','.join(output)
```

Comment:
Why this? Pandas doesn't take a list of strings ?

Reply:
It's because mapInPandas cannot pickle a tuple to return it to the driver.

Reply:
Are we sure the name can never have a comma? Should we use a more exotic unicode character to reduce collisions?

Reply:
hmm, good point. Any suggestion what unicode character to use?